### PR TITLE
Merge python-aptly-gui with python-aptly

### PR DESCRIPTION
**Pull request by chnyda**
Add aptly-gui (pyqt5) to python-aptly

### CODE REVIEW

File: aptlygui/views/list_tab.py [hunk 1]
```diff
@@ -0,0 +1,54 @@
+#!/usr/bin/env python3
+
+from PyQt5.QtCore import Qt
+from PyQt5.QtGui import QStandardItemModel
+from PyQt5.QtWidgets import (QGridLayout, QFormLayout, QLabel, QPushButton, QWidget, QListView, QAbstractItemView,
+                             QVBoxLayout)
+
+
+class ListTab(QWidget):
```
Comment (fpytloun) [inline aptlygui/views/list_tab.py]: This should probably go to aptlygui.widgets?

**Comment by chnyda** [comment]
Done, @fpytloun

**Comment by fpytloun** [comment]
Hmm, I am thinking if it's really good idea to join aptly-gui with python-aptly or if it won't be easier to keep it separate at the end (for packaging, etc. as installing python-aptly would also install pyqt5 and things that you typically don't need on a server)..
BTW don't you have Jabber, IRC or something to connect in real time? :-)

**Comment by chnyda** [comment]
I agree that systematically installing qt is not ideal. Let's keep python-aptly and its gui separated. I'm available on IRC.

**Pull request closed without merging**